pages/common/mysqld.md

```diff
@@ -0,0 +1,32 @@
+# mysqld
+
+> Start the MariaDB database server.
+> More information: <https://dev.mysql.com/doc/refman/8.0/en/mysqld.html>.
+
+- Start the MariaDB database server:
+
+`mysqld`
+
+- Display error messages:
```

Comment:
This option is [Windows only](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_console), so:
```suggestion
- Start the database server, printing error messages to the console (Windows only):
```

Reply:
`Write error output on screen; don't remove the console window on windows.`

I think that means that on linux and macos, errors get printed in the console and in windows the console deosn't get closed as it normally would. But maybe I'm misinterpreting it.

Reply:
Where'd you get that? I can't find it :/

Reply:
That's somewhere in the 3000 lines of the help page, when you run `mysqld --verbose --help` :sweat_smile:. 
![image](https://user-images.githubusercontent.com/50295997/111217180-93da0800-85d5-11eb-903d-f13a2fcdbccd.png)

Reply:
Okay then, not sure what the docs are talking about 🤷🏻 suggestion updated.